front/lib/connectors_api.ts

```diff
@@ -238,15 +260,26 @@ async function _resultFromResponse<T>(
       logger.error({ jsonError }, "Unexpected response from ConnectorAPI");
       return new Err(jsonError);
     } else {
-      logger.error(
-        { statusCode: response.status, statusText: response.statusText },
-        "Unexpected response from ConnectorAPI"
-      );
-      return new Err({
-        error: {
-          message: `Unexpected response status: ${response.status} ${response.statusText}`,
-        },
-      });
+      try {
+        const textError = await response.text();
+        const errorResponse = JSON.parse(textError);
+        logger.error(
```

Comment:
The error is not unexpected here is it?

Reply:
We were logging + returning an error always in this function, for consistency I kept this behavior.
I can remove the logs if that's overkill

Reply:
Well the log is fine but incorrect. The error is not yet unexpected here, it's expected since we can unpack it :)

Reply:
True! I removed!